pkg/reconcilermanager/controllers/rootsync_controller.go

```diff
@@ -651,21 +652,21 @@ func (r *RootSyncReconciler) mapSecretToRootSyncs(secret client.Object) []reconc
 	}
 
 	attachedRootSyncs := &v1beta1.RootSyncList{}
-	listOps := &client.ListOptions{
-		FieldSelector: fields.OneTermEqualSelector(gitSecretRefField, sRef.Name),
-		Namespace:     sRef.Namespace,
-	}
-	if err := r.client.List(context.Background(), attachedRootSyncs, listOps); err != nil {
+	if err := r.client.List(ctx, attachedRootSyncs, client.InNamespace(sRef.Namespace)); err != nil {
```

Comment:
Could you pull out these snippets for RootSync/RepoSync secret mapping to a separate change? They seem unrelated to the controller registration.

Also, why is this preferred to the field selector?

Reply:
I suppose I could remove the indexes as a separate PR, but it's only really required because of this PR.

Would that make it easier to review?